api/service/staking/service.go

```diff
@@ -47,19 +47,21 @@ type Service struct {
 	stopChan      chan struct{}
 	stoppedChan   chan struct{}
 	accountKey    *ecdsa.PrivateKey
+	blsAddress    [20]byte
```

Comment:
I don't quite understand how blsAddress/blsKey related to the ecdsa private key. Do you plan to use the same private key?

Reply:
No, The BLS and ECDSA address doesn't have to be from the same private key. This is just to bind the staking to a specific BLS address so the concensus can work with the staking.